components/org.wso2.carbon.identity.query.saml/src/test/java/org/wso2/carbon/identity/query/saml/validation/TestUtil.java

```diff
@@ -0,0 +1,204 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.identity.query.saml.validation;
+
+import org.opensaml.core.xml.util.XMLObjectChildrenList;
+import org.opensaml.saml.saml2.core.*;
+import org.opensaml.saml.saml2.core.impl.*;
+
+import org.wso2.carbon.base.CarbonBaseConstants;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * Utilclasses for testcases
+ */
+public class TestUtil {
+
+
+    public static void initPrivilegedCarbonContext(String tenantDomain, int tenantID, String userName) {
+
+        String carbonHome = Paths.get(System.getProperty("nuser.dir"), "target").toString();
+        System.setProperty(CarbonBaseConstants.CARBON_HOME, carbonHome);
+        PrivilegedCarbonContext.startTenantFlow();
+        PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantDomain(tenantDomain);
+        PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantId(tenantID);
+        PrivilegedCarbonContext.getThreadLocalCarbonContext().setUsername(userName);
+    }
```

Comment:
where's the end tenant flow? It should also be a util method.

Reply:
Okay,ill create it

Reply:
thanks.